### PR TITLE
IOS-Chat : Fixes for chat history 

### PR DESCRIPTION
**Pull request by mayuri-talkshoplive**
- Passed start timetoken by default
- Sorted final messages

### CODE REVIEW

File: Sources/Talkshoplive/Core/Chat/ChatProvider.swift [hunk 1]
```diff
@@ -438,6 +439,15 @@ public class ChatProvider {
                         // Notify when all message processing is complete
                         dispatchGroup.notify(queue: .main) {
                            Config.shared.isDebugMode() ? print("History : Fetched successfully!") : ()
+                            //Sort data according to published timeinterval
+                            messageArray.sort(by: {
+                                guard let publishedFirst = $0.published, let timeIntervalFirst = Int(publishedFirst),
```
Comment (daman-talkshoplive) [inline Sources/Talkshoplive/Core/Chat/ChatProvider.swift:444]: Messages should be in desc order by default when you pass the start timetoken.

File: Sources/Talkshoplive/Utils/Helpers/Date_Extension.swift [hunk 2]
```diff
@@ -9,7 +9,7 @@ import Foundation
 
 extension Date {
     var milliseconds: TimeInterval {
-        return self.timeIntervalSince1970 * 1000.0
+        return self.timeIntervalSince1970 * 10000000
```
Comment (daman-talkshoplive) [inline Sources/Talkshoplive/Utils/Helpers/Date_Extension.swift:12]: Milliseconds should be x 1000. This looks more of nano seconds?

Comment (mayuri-talkshoplive) [reply Sources/Talkshoplive/Utils/Helpers/Date_Extension.swift:12]: @daman-talkshoplive  If I pass with * 1000 , it's not working , it's giving me blank array.

File: Sources/Talkshoplive/Core/Chat/ChatProvider.swift (hunk 1 above)
Comment (mayuri-talkshoplive) [reply Sources/Talkshoplive/Core/Chat/ChatProvider.swift:444]: @daman-talkshoplive it's coming in proper order , confirmed with front end as well.

File: Sources/Talkshoplive/Utils/Helpers/Date_Extension.swift (hunk 2 above)
Comment (mayuri-talkshoplive) [reply Sources/Talkshoplive/Utils/Helpers/Date_Extension.swift]: @daman-talkshoplive yes it's takes nanoseconds. Fixed for function naming conversation.

**Review by daman-talkshoplive — APPROVED** [review]

**Pull request merged**